aspnetcore/release-notes/aspnetcore-10/includes/MinimalAPIValidationRecordTypes.md

```diff
@@ -1,7 +1,21 @@
-### Improvements to Minimal APIs Validation
-
+#### Validation with record types
 <!-- https://github.com/dotnet/aspnetcore/pull/61193 -->
 <!-- https://github.com/dotnet/aspnetcore/pull/61402 -->
-Beginning in Preview 4, ASP.NET Core supports validation on record types. With this enhancement,
-inputs to API endpoints defined using `record` types are now validated automatically, in the same way as
-class types.
\ No newline at end of file
+
+Minimal APIs also support validation with C# record types. Record types can be validated using attributes from the [`DataAnnotations`](xref:System.ComponentModel.DataAnnotations) namespace, similar to classes. For example:
+
+```csharp
+public record Product(
+    [Required] string Name,
+    [Range(1, 1000)] int Quantity);
+```
+
+When using record types as parameters in minimal API endpoints, validation attributes are automatically applied in the same way as class types:
```

Comment:
I thought everyone was capitalizing "M" in "Minimal API," but I might be mistaken. All of the Blazor docs do it that way.

```suggestion
When using record types as parameters in Minimal API endpoints, validation attributes are automatically applied in the same way as class types:
```